application/src/main/data/json/system/widget_types/rpc_remote_shell.json

```diff
@@ -11,7 +11,7 @@
termInfo.cwd) {\n                cwd = termInfo.cwd;\n            }\n            if (callback) {\n                callback(termInfo);\n            }\n        },\n        function() {\n            if (callback) {\n                callback(null);\n            }\n        }\n    );\n}\n\nfunction sendCommand(terminal, command) {\n    terminal.pause();\n    var sendCommandRequest = {\n        command: command,\n        cwd: cwd\n    };\n    self.ctx.controlApi.sendTwoWayCommand('sendCommand', sendCommandRequest, requestTimeout).subscribe(\n        function (responseBody) {\n            if (responseBody && responseBody.ok) {\n                commandExecuting = true;\n                setTimeout(pollCommandStatus.bind(null, terminal), commandStatusPollingInterval);\n            } else {\n                var error = responseBody ? responseBody.error : 'Unhandled error.';\n                terminal.error(error);\n                terminal.resume();\n            }\n        },\n        function () {\n            onRpcError(terminal);\n        }\n    );\n}\n\nfunction terminateCommand(terminal) {\n    self.ctx.controlApi.sendTwoWayCommand('terminateCommand', null, requestTimeout).subscribe(\n        function (responseBody) {\n            if (!responseBody.ok) {\n                commandExecuting = false;\n                terminal.error(responseBody.error);\n                terminal.resume();\n            }\n        },\n        function () {\n            onRpcError(terminal);\n        }\n    );\n}\n\nfunction onRpcError(terminal) {\n    var errorText = self.ctx.defaultSubscription.rpcErrorText;\n    terminal.error(errorText);\n    terminal.resume();\n}\n\nfunction pollCommandStatus(terminal) {\n    self.ctx.controlApi.sendTwoWayCommand('getCommandStatus', null, requestTimeout).subscribe(\n        function (commandStatusResponse) {\n            for (var i = 0; i < commandStatusResponse.data.length; i++) {\n                var dataElement = commandStatusResponse.data[i];\n                if (dataElement.stdout) {\n                    terminal.echo(dataElement.stdout);\n                }\n                if (dataElement.stderr) {\n                    terminal.error(dataElement.stderr);\n                }\n            }\n            if (commandStatusResponse.done) {\n                commandExecuting = false;\n                cwd = commandStatusResponse.cwd;\n                terminal.resume();\n            } else {\n                var interval = commandStatusPollingInterval;\n                if (!commandStatusResponse.data.length) {\n                    interval *= 5;\n                }\n                setTimeout(pollCommandStatus.bind(null, terminal), interval);\n            }\n        },\n        function () {\n            commandExecuting = false;\n            onRpcError(terminal);\n        }\n    );\n}\n\nself.onResize = function () {\n    if (terminal) {\n        terminal.resize(self.ctx.width, self.ctx.height);\n    }\n};\n\nself.onDestroy = function() {\n};",
```

Comment:
`if (command.trim() === 'exit') {
                        if (!simulated) {
                            self.ctx.controlApi.sendTwoWayCommand('sendCommand', {
                                command: 'exit',
                                cwd: cwd
                            }, requestTimeout).subscribe();
                        }
                        this.disable();
                        return;`